components/rhs_thread/rhs_thread.tsx

```diff
@@ -172,6 +172,19 @@ export default class RhsThread extends React.Component<Props, State> {
         }
     }
 
+    private userIsAtLastPost = () => {
+        const postReplies = (this.rhspostlistRef.current as HTMLElement)?.childNodes;
+        const rhsPostListRect = (this.rhspostlistRef.current as HTMLElement)?.getBoundingClientRect();
+        const offset = 24;
+
+        if (postReplies) {
+            const postRepliesArray = Array.from(postReplies);
+            return ((postRepliesArray[postRepliesArray.length - 1] as HTMLElement).offsetTop + offset) + rhsPostListRect.top === rhsPostListRect.bottom;
+        }
+
+        return false;
```

Comment:
if there are no replies, shouldn't it be true that user is at last post?

Reply:
Yes

Reply:
Sorry actually no, in this case we should keep it false, otherwise it will scroll every time, since it's in an `||` condition in the scrollToBottom logic